Bottom: fab5a69f162c380d6cfcf4afdb4b816ac48e1ed5
Top:    ebd12a3c7f7782c2f293dede62fd215fa644327d
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-12 20:45:59 +0200

Refresh of add-a-calculator-model-object

---

diff --git a/src/modules/calculator/CMakeLists.txt b/src/modules/calculator/CMakeLists.txt
index 089f030..683fd89 100644
--- a/src/modules/calculator/CMakeLists.txt
+++ b/src/modules/calculator/CMakeLists.txt
@@ -8,7 +8,9 @@ target_compile_options(calculator
 )
 add_module(calculator.model model.cpp)
 add_module(calculator.value value.cpp)
+add_dependencies(calculator.model.pcm
+	calculator.value.pcm
+)
 add_dependencies(calculator
 	calculator.model.pcm
-	calculator.value.pcm
 )
